Tests/test_bmp_reference.py

```diff
@@ -40,6 +40,7 @@ def test_questionable():
         "rgb32fakealpha.bmp",
         "rgb24largepal.bmp",
         "pal8os2sp.bmp",
+        "pal8rletrns.bmp",
```

Comment:
When running this locally, I was told to add this by the test.
https://github.com/python-pillow/Pillow/blob/38f4660d9ecfab74af8bad292d31e080072b25b3/Tests/test_bmp_reference.py#L50